Source/ActionController.swift

```diff
@@ -179,13 +179,29 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
             section.actions.append(action)
         }
     }
-
     @discardableResult
     open func addSection(_ section: Section<ActionDataType, SectionHeaderDataType>) -> Section<ActionDataType, SectionHeaderDataType> {
         _sections.append(section)
         return section
     }
     
+    public typealias ClosureAction = ((Action<ActionDataType>) -> Bool)
+    
+    open func removeAction(closure: ClosureAction) {
```

Comment:
I would use `where` for the argument label as it looks more like the functions provided by the array class. What do you think?